app/forms/assessor_interface/age_range_subjects_form.rb

```diff
@@ -69,13 +69,25 @@ def permittable_parameters
   private
 
   def update_age_range
-    note = find_or_create_note(assessment.age_range_note, age_range_note)
+    if age_range_note
```

Comment:
I think this'll mean the `note` variable isn't defined if `age_range_note` is false, and then the line below will fail to find the variable. It might make more sense to transform the `age_range_note` variable, something like this:

```rb
age_range_note = "Age range changed: #{age_range_note}" if age_range_note.present?
```

And then the call to `find_or_create_note` would be unchanged in this PR.